pajbot/managers/redis.py

```diff
@@ -17,7 +18,7 @@ class RedisManager:
 
     @staticmethod
     def init(**options):
-        default_options = {"decode_responses": True}
+        default_options = {"decode_responses": True, "host": os.environ.get("REDIS", "localhost")}
```

Comment:
There's already a `[redis]` configuration section in the config, is it possible to move the docker-specifica there?

It would just be:

```none
[redis]
host=a.b.c.d
```

Reply:
I could not find  the redis in the `config.example.ini`, and i was not aware how i could easily access the config files from other classes.
Also since the containers are running on an internal network, instead of connecting to a local ip `a.b.c.d`, they should connect to the `name` of the service requested, as seen in these types of environment variables can be seein in the docker-compose file.

Reply:
Also, since the `redis`, and `twitch-api-v3-proxy`, URI's have to be the same for all bot's and web's would it not be smarter to use something like environment variables in the docker-compose file?

Reply:
Nothing wrong with environment variables, it's just that are not used at all in the bot currently. I might look into this further (maybe we can implement a shared config) because docker deployment is something i support and want to see, and I think the same applies for @pajlada.

Reply:
The thing is, those items i have choose to use environment variables for, are settings which should be the same for all the `bot`'s and `web`'s defined, i can see the use for using the config where settings have to differ, like the `mysql` connection, where each of them have to use a different database.
But as far as i understood from spamming pajlada, all the current bots and webs use the same `redis` databse and `twitch-api-v3-proxy` instance?

Reply:
Yea, they all share it.